projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -30,22 +39,24 @@ import {
   SortDirection
 } from './index';
 import { sortBy } from './go-table-utils';
-import { fadeTemplateAnimation } from '../../animations/fade.animation';
-import { detailButtonAnim, tableRowBorderAnim } from '../../animations/table-details.animation';
 import { GoTablePage } from './go-table-page.model';
 
 @Component({
   animations: [
-    detailButtonAnim,
     tableRowBorderAnim,
     fadeTemplateAnimation
   ],
   selector: 'go-table',
   templateUrl: './go-table.component.html',
   styleUrls: ['./go-table.component.scss']
 })
-export class GoTableComponent implements OnInit, OnChanges, AfterViewInit {
+export class GoTableComponent implements OnInit, OnChanges, OnDestroy, AfterViewInit {
 
+  /**
+   * Used to dynamically access the child rows on each row
+   * item when implementing the goTableChildRows template
+   */
+  @Input() childRowsKey: string = null;
```

Comment:
This is the key to accessing the children of the row I referred to.

We cannot assume that the child property of the data will be `children` for every set of data. This allows the implementor to specify the key of the property keeping the logic dynamic for every implementor.